src/test/java/unit/WorkspaceUser.java

```diff
@@ -154,14 +165,26 @@ private static void expectListedUserWithRoles(String userEmail, IamRole... roles
 
   /**
    * Helper method to call `terra workspace list` and filter the results on the specified user
-   * email.
+   * email. Uses the current workspace.
    */
-  private static Optional<UFWorkspaceUser> listWorkspaceUserWithEmail(String userEmail)
+  static Optional<UFWorkspaceUser> listWorkspaceUserWithEmail(String userEmail)
+      throws JsonProcessingException {
+    return listWorkspaceUserWithEmail(userEmail, null);
+  }
+
+  /**
+   * Helper method to call `terra workspace list` and filter the results on the specified user email
+   * and workspace id (uses the current workspace id if null).
+   */
+  static Optional<UFWorkspaceUser> listWorkspaceUserWithEmail(String userEmail, UUID workspaceId)
       throws JsonProcessingException {
     // `terra workspace list-users --format=json`
     List<UFWorkspaceUser> listWorkspaceUsers =
-        TestCommand.runAndParseCommandExpectSuccess(
-            new TypeReference<>() {}, "workspace", "list-users");
+        workspaceId == null
+            ? TestCommand.runAndParseCommandExpectSuccess(
```

Comment:
maybe make an overload so you don't have to declare an empty `TypeReference`.

Reply:
This isn't really empty, it's just Java's new type inference that fills in the type behind the scenes using the return value.